apps/console/src/features/connections/pages/connection-templates.tsx

```diff
@@ -33,6 +33,7 @@ import {
     SearchWithFilterLabels,
     useDocumentation
 } from "@wso2is/react-components";
+import { identityProviderConfig } from "apps/console/src/extensions";
```

Comment:
Let's import from the relative paths

Reply:
https://github.com/wso2/identity-apps/pull/4802/commits/8b521dbe39a2b42cb4e246fa508f14fba17a8834